### PR TITLE
docs(aio): adjust zips/plunker paths

### LINKED ISSUE #16117: docs(aio): Some plunkers and zips needs the url to be adapted
For example ts-to-js, cli-quickstart and webpack

### PR DESCRIPTION
**Pull request by Foxandxss**
Fixes #16117

### CODE REVIEW

**Comment by Foxandxss** [comment]
Hold on this one, there is a weird behavior.

File: aio/content/guide/cli-quickstart.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ You'll pursue these ends in the following high-level steps:
 3. [Serve](guide/cli-quickstart#serve) the application.
 4. [Edit](guide/cli-quickstart#first-component) the application.
 
-And you can also <a href="/resources/zips/cli-quickstart/cli-quickstart.zip">download the example.</a>
+And you can also <a href="/content/zips/cli-quickstart/cli-quickstart.zip">download the example.</a>
```
Comment (gkalpak) [inline aio/content/guide/cli-quickstart.md]: This is trying to fetch `/content/docs/content/zips/cli-quickstart/cli-quickstart.zip.json`.
I think you need to tell the LocationService that it should not try to navigate to this as a doc, by either adding a `download` attribute or a `target` (other than `_self`).

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/guide/ts-to-js.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ can read and write Angular apps in their preferred dialect.
 * [View and Child Decorators](guide/ts-to-js#view-child-decorators)<br>
 * [AOT compilation in _TypeScript_ Only](guide/ts-to-js#aot)<br>
 
-**Run and compare the live <live-example name="cb-ts-to-js">TypeScript</live-example> and <live-example name="cb-ts-to-js" lang="js">JavaScript</live-example>
+**Run and compare the live <live-example name="cb-ts-to-js/ts">TypeScript</live-example> and <live-example name="cb-ts-to-js/js">JavaScript</live-example>
```
Comment (gkalpak) [inline aio/content/guide/ts-to-js.md]: The zips for these files cannot be found on the preview.

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/tools/example-zipper/exampleZipper.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ class ExampleZipper {
     let exampleZipName;
 
     if (relativeDirName.indexOf('/') !== -1) { // Special example
-      [relativeDirName, exampleZipName] = relativeDirName.split('/');
+      exampleZipName = relativeDirName.split('/')[0];
```
Comment (gkalpak) [inline aio/tools/example-zipper/exampleZipper.js]: Could the zip name be `relativeDirName.split('/').join('-')` or something? I think it is useful to not have the same name for different zips.

**Review by gkalpak — APPROVED** [review]
One minor suggestion. LGTM regardless.

File: aio/tools/example-zipper/exampleZipper.js (hunk 3 above)
Comment (Foxandxss) [inline aio/tools/example-zipper/exampleZipper.js]: I don't think there is any case where that would happen.

Comment (gkalpak) [inline aio/tools/example-zipper/exampleZipper.js]: It does happen with the `cb-ts-to-js` one, but it shipped now, so...

Comment (petebacondarwin) [inline aio/tools/example-zipper/exampleZipper.js]: Can you create an issue to track it @gkalpak

Comment (gkalpak) [inline aio/tools/example-zipper/exampleZipper.js]: I can...but will I? 😛

Comment (gkalpak) [inline aio/tools/example-zipper/exampleZipper.js]: For reference: #16227

**Pull request merged**